fix/ODD773 Add Google Analytics tracking code

chuanlin2018 opened the pull request:
http://mml.nist.gov:8080/browse/ODD-773

The reason that SDP didn't "phone home" was that I didn't add that piece of Google Analyst code to index.html. I also added the code to help, about and policy page (the search page already has it).

This should make it work as before. But it's not fully functioning with SPA. When navigating to search, about, policy or help page, GA function will not triggered unless user refresh the pages. This will be fixed once I figure out how to use the GA function provided by DAP and apply it to router event.

This review just need to confirm that when open SDP home page, or you refresh search, about, policy or help page, "analytics.js" should show up in "Network" tab in the Chrome Developer Tools.

RayPlante commented:
If you put the `<script>` element in `index.html` _and_ the different template files, won't it in effect appear twice in the final page?  We should chat about this a bit.

chuanlin2018 commented:
You are right! I just tested. The app behaved the same after I removed the script from individual pages. Still though, have to refresh the pages to trigger the script.

Updated and all checks passed.
